servicecontrol/capacity-and-planning.md

```diff
@@ -61,4 +61,31 @@ The transports supported by ServiceControl out-of-the-box (i.e. MSMQ, RabbitMQ,
+
+Making decisions about the use of the other plugins and features requires a little more thought to balance the smooth running of your system with your actual requirements. The temptation to just use them all as a kind of insurance policy is not a good choice. The cost of having some of these features in play can actually cause issues down the track if you're not prepared.
+ 
+[Auditing](/nservicebus/operations/auditing.md) is a good example of how the cost of having a record of everything may cause significant impact on the system. It might be worth considering turning audit off in some or all endpoints in production. Or if you absolutely need it move the auditing to another place like your Business Intelligence database or somewhere that has better resources dedicated to the problem.
+
+It is technically possible to run two instances of ServiceControl one for audits and one for active monitoring and retries. Whether this makes sense for your situation is another discussion entirely.
+
+[Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) and [Custom Checks](/servicepulse/intro-endpoints-custom-checks.md) are great for knowing when an endpoint is up or down but they add extra noise to your system. Definitely think hard about what the business ramifications are of having each endpoint available. Often, not all endpoints are mission critical, the default heartbeat is forty seconds. Maybe ten second updates are right for some endpoints but two minute intervals could be better for others. Remember also that all communication from endpoints with ServiceControl is performed via messaging. Adding a message to the queue every second may have little impact on when the notification shows up in ServicePulse. Remember, the last heartbeat is really the only one you care about. If you stop ServiceControl those heartbeats are going to bank up in the queue. You might want to consider flushing the heartbeats from the queue before starting ServiceControl again. Having a lot of noisey messages that add no value may overwhelm ServiceControl and causing it fail. 
+
+##### Sometimes slower is better.
+
+Moving forward pace your adoption of plugins and features: 
+
+- Turn off auditing on all endpoints as well as heartbeats and custom checks.
+- If you are using the NServiceBus.Host set the log levels to ['production'](/nservicebus/hosting/nservicebus-host/profiles.md#logging-behaviors)
+- Perform a load test to baseline your solution.
+- When you are comfortable with the performance of the system try adding Heartbeats. That will allow you to monitor your system
```

Comment:
"That will allow you to monitor your system"... is this missing some trailing words?